packages/components/src/components/popover/popover.jsx

```diff
@@ -54,7 +54,7 @@ const Popover = ({
     return (
         <div ref={hover_ref} className={classNames({ 'dc-popover__wrapper': relative_render })}>
             {relative_render && (
-                <div className='dc-popover__container' style={{ zIndex: zIndex || 1 }}>
+                <div className='dc-popover__container' style={{ zIndex }}>
```

Comment:
IMO better to either pass the `zIndex` value as a prop or increase the default value here to `5` rather than ignoring the prop value. 
zIndex is being passed from so many different places that we cannot really hardcode it to `1000`.

Reply:
I think we need a system for handling our layers on z axis since we have this values for z-index: 
-2,-1, 0,1, 2,3, 4, 5, 10, 11,12,14, 20, 30,40, 50,99, 100, 107, 999,1000, 9999, 20000 , 1000, 100000, 9999999

Here since it's a Popover component and we always need it to be on top we should use the highest number in the z layer.  I assume it won't hurt other places, but to be sure changed it to passing where it's necessary for now

Reply:
@hojjat-fs IMO it is better to revert zIndex default value `1`, and most of the zIndex added to other component and only fix the issue with the footer popover.
Adding zIndex to other components may need more QA session or bring up other issues.